src/scenario.cpp

```diff
@@ -547,9 +547,15 @@ time_point scenario::start_of_game() const
 {
     time_point ret;
 
+    int options_start_hour = get_option<int>( "INITIAL_TIME" );
+    bool override_start_hour = options_start_hour == -1;
```

Comment:
No need for adding this new variable. You can check for `options_start_hour` directly.

Reply:
Ok, I cleaned that up.